src/Modules.js

```diff
@@ -31,12 +32,12 @@ module.exports = {
   'ndvi-colormap': require('./modules/NdviColormap'),
   'paint-bucket': require('./modules/PaintBucket'),
   'overlay': require('./modules/Overlay'),
-  'replace-color':require('./modules/ReplaceColor'),
+  'replace-color': require('./modules/ReplaceColor'),
   'resize': require('./modules/Resize'),
   'rotate': require('./modules/Rotate'),
   'saturation': require('./modules/Saturation'),
   'text-overlay': require('./modules/TextOverlay'),
   'threshold': require('./modules/Threshold'),
   'tint': require('./modules/Tint'),
-  'color-temperature': require('./modules/ColorTemperature')
+  'canvas-resize': require('./modules/CanvasResize')
```

Comment:
@tech4GT may be you can add it in alpabetical order. So that the list will not become unordered.

Reply:
Hi @Harshithpabbati Aren't we sorting these anyway?
I mean earlier the color-temperature was way at the end but it shows up just fine in the UI.

Reply:
Oh yeah I thought we weren't sorting it. Sorry.